src/component/graph/Graph.ts

```diff
@@ -12,8 +12,49 @@ mxGraph.prototype.edgeLabelsMovable = false;
 mxGraph.prototype.cellsLocked = true;
 mxGraph.prototype.cellsSelectable = false;
 
+class MySvgCanvas2D extends mxSvgCanvas2D {
+  public minStrokeWidth: number;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  constructor(root: any, styleEnabled?: any) {
+    super(root, styleEnabled);
+  }
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  addNode(filled: any, stroked: any): void {
+    super.addNode(filled, stroked);
+  }
+}
+
+mxShape.prototype.createSvgCanvas = function() {
```

Comment:
:question: are we sur we want to use the canvas even in production (ie not only when testing)?
If so I suggest to put the mxShape prototype configuration elsewhere for a better separation of concernts (like described in https://github.com/bonitasoft-labs/bpmn-visu-js/pull/118#discussion_r394665895)